mage_ai/frontend/components/DataTable/index.tsx

```diff
@@ -227,6 +235,11 @@ function Table({
             cellStyle.width = maxWidthOfFirstColumn;
           }
 
+          if (isInvalid) {
+            cellStyle.background = light.interactive.dangerBorder;
```

Comment:
This isn't the preferred way to style our components, hardcoding an inline style. It might work, but we should be using a React styled component with props.

Reply:
I'll see if I can do something else.